src/sentry/api/endpoints/organization_codeowners_associations.py

```diff
@@ -9,7 +9,7 @@
 
 
 class OrganizationCodeOwnersAssociationsPermission(OrganizationPermission):
-    scope_map = {"GET": ["org:integrations"]}
+    scope_map = {"GET": ["org:read", "org:write", "org:integrations"]}
```

Comment:
I don't see `org:admin` listed here; does this mean that admins cannot make changes to code owners?

Reply:
this one was weird, maybe we should re-use the same permissions